pkg/models/model.go

```diff
@@ -26,6 +26,7 @@ import (
 )
 
 type Model struct {
+	Topology       string          `yaml:"topology"`
```

Comment:
Why do we need Topology field here?

Reply:
The reason is in the GetTestProvider() function in pkg/factory/provider.go. In the case of using a specific model file, which gets imported to file, that tree has to have a metadata tag so that the ToGraph() function can be called on line 82 (if it doesn't, one of the changes is that an error is thrown). So the "Topology" field is necessary in the model file so that the model can be correctly tagged in this context.

Reply:
Topology field removed, the model import will now set to block topology if an nvlink domain is specified.